examples/full_stack/README.md

```diff
@@ -0,0 +1,102 @@
+# embedchain-ui
```

Comment:
Let's clean up this README and only add instructions on how to run this demo. We don't want to overwhelm the user by all the content in README. 😅